src/test/java/fi/otavanopisto/kuntaapi/test/AbstractIntegrationTest.java

```diff
@@ -90,11 +94,34 @@ public void afterEveryTest() throws IOException {
     deleteSystemSettings();
   }
   
+  private void deleteIndices() {
+    try {
+      String types = "service";
+      String body = "{\"query\": {\"match_all\": {} } }";
+      HttpPost httpPost = new HttpPost(String.format("http://localhost:9200/kunta-api/%s/_delete_by_query?conflicts=proceed", types));
+      httpPost.setEntity(new StringEntity(body));
+      
+      CloseableHttpClient httpClient = HttpClients.createDefault();
+      try {
+        httpClient.execute(httpPost);
+      } finally {
+        httpClient.close();
+      }
+    } catch (IOException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)